studies/models.py

```diff
@@ -315,74 +352,41 @@ class Study(models.Model):
     )
     negativity = models.CharField(
         _(
-            "Bevat bovenstaand onderzoekstraject elementen die \
-<em>tijdens</em> de deelname niet-triviale negatieve emoties kunnen opwekken? \
-Denk hierbij bijvoorbeeld aan emotioneel indringende vragen, kwetsende \
-uitspraken, negatieve feedback, frustrerende, zware, (heel) lange en/of \
-(heel) saaie taken."
-        ),
-        max_length=1,
-        choices=YesNoDoubt.choices,
-        blank=True,
-    )
-    negativity_details = models.TextField(_("Licht je antwoord toe."), blank=True)
-    stressful = models.CharField(
-        _(
-            "Bevat bovenstaand onderzoekstraject elementen die tijdens de \
-deelname zodanig belastend zijn dat deze <em>ondanks de verkregen \
-informed consent</em> vragen zou kunnen oproepen (of zelfs \
-verontwaardiging), bijvoorbeeld bij collega-onderzoekers, bij de deelnemers \
-zelf, of bij ouders of andere vertegenwoordigers?"
-        ),
-        help_text=mark_safe_lazy(
-            _(
-                "Dit zou bijvoorbeeld het geval kunnen zijn \
-bij een 'onmenselijk' lange en uitputtende taak, een zeer confronterende \
-vragenlijst, of voortdurend vernietigende feedback, maar ook bij een ervaren \
-inbreuk op de privacy, of een ander ervaren gebrek aan respect. \
-Let op, het gaat bij deze vraag om de door de deelnemer ervaren belasting \
-tijdens het onderzoek, niet om de opgelopen psychische of fysieke schade \
-door het onderzoek."
-            )
+            "Bevat bovenstaand onderzoekstraject elementen die <em>tijdens\
+            </em> de deelname zodanig belastend zijn dat deze vragen, \
```

Comment:
I don't know if this is a conscious change, or something that Black took liberties with. But this new whitespace is actually significant.

A line with a trailing `\` is continued in a very low-level manner, without the sanity checks Python usually provides. The interpreter literally sees the next line as a the same line and that leading indentation whitespace gets included in the string. That's why these msgstrs have loads of whitespace in django.po now. 

Somehow gettext doesn't mind (they're not fuzzy or anything) but this looks pretty awful:
![image](https://github.com/DH-IT-Portal-Development/ethics/assets/7064137/518c25da-19e5-407d-bd76-60a0735b426b)

The recommended way to continue strings is to just do it implicitly:

```python

    negativity = models.CharField(
        _(
            "Bevat bovenstaand onderzoekstraject elementen die <em>tijdens"
            "</em> de deelname zodanig belastend zijn dat deze vragen, "
            "weerstand, of zelfs verontwaardiging zouden kunnen oproepen, "
            "bijvoorbeeld bij collega-onderzoekers, bij de deelnemers zelf, "
            "of bij ouders of andere vertegenwoordigers?"
        ),
        max_length=1,
        choices=YesNoDoubt.choices,
        blank=True,
    )
```

Just keep adding strings without commas inbetween and python will implicitly join them. Or use triple quotes, `str.join()`, or literally anything other than backslashes, lol.

A lot of the strings here were already continued with backslashes from ages ago, which might not have been such bad practice back then. I wouldn't ask you to fix all of them just try to keep the strings you change free of unnecessary whitespace.